### ISSUE TITLE
Disable Audio While Seeking

### ISSUE THREAD

**Issue by April-June-August** [iina/iina#4603]
**System and IINA version:**

- macOS 
macOS 13.5.1
- IINA 
1.3.3 Build 138

**Expected behavior:**
Audio won't be play-backed while performing seek or "Next Frame" or "Previous Frame".

**Actual behavior:**
Fragments of meaningless audio played while performing seek or "Next Frame" or "Previous Frame".

**Steps to reproduce:** Play a video with audio, perform seek or "Next Frame" or "Previous Frame". 

- [x] MPV does not have this problem.

**How often does this happen?**

Always.

**Comment by low-batt** [comment]
Reproduced for me with both IINA and mpv.

This is mpv issue https://github.com/mpv-player/mpv/issues/6104.

**Comment by April-June-August** [comment]
That's weird, mpv kept silent when I was seeking the video but IINA didn't. 🤔️

**Comment by low-batt** [comment]
Always troubles me when someone reports my analysis doesn't match up with what they are experiencing. Usually means there is more to the situation and I missed something.

I did not dig into seeking behavior. I paused mpv playback and tested `frame-step` by pressing the period key. That definitely was playing short bursts of audio. This is with the internal speakers on my MacBook. If you are using wireless speakers that might behave differently.

Anyway, that got me looking at mpv and I found the above issue. We will need to monitor that issue and see how mpv addresses it.

**Comment by AaronKelley** [comment]
Noticing that upstream (see mpv issue linked above), there is now a flag to mute audio during a frame skip. Can this be applied to IINA?

**Comment by low-batt** [comment]
YES. That is one of the changes that needs to be made to IINA when we upgrade mpv again. That will happen in IINA 1.5.0. We started building upgraded libraries for 1.5.0 and ran into problems building for Intel Macs. I suspect there is a mpv regression that needs to be fixed, but I have not had time to confirm that.

Because of the problems we have put 1.5.0 development on the side for the moment and are now working on a quick 1.4.2 release to fix some more regressions before moving back to 1.5.0.
